core/src/components/refresher/refresher.tsx

```diff
@@ -59,6 +59,15 @@ export class Refresher implements ComponentInterface {
    */
   @Prop() snapbackDuration = '280ms';
 
+  /**
+   * Increase or decrease the pixels who are moved by cursor on pull down
+   * Defaults to `100`, the pulled down pixels are equal to the cursor
+   *
+   * Example: If you increase this value to `120`, instead of 10 Pixel it will be 12 pixel pulled down (increase of 20 percent)
+   * If you decrease to `80`, the pull down will be less than the moved pixels of the cursor
+   */
+  @Prop() pullDownFactor = 100;
```

Comment:
So I think we should keep the naming consistent with the others and just use `pullFactor`. This is because if we were to ever add support for refreshing from the bottom it wouldn't make sent to call it "pull down". 🙂

Reply:
Would it make sense to support setting a % as well as a pixel value? Having `pullFactor` as a number makes me think it's a pixel value, when in reality the pixel value appears to be 10% of `pullFactor`. If not, I would suggest making it very clear that the value is supposed to represent a % instead of a pixel value.

Reply:
Also, the docstring doesn't really make any sense:

```
Increase or decrease the pixels who are moved by cursor on pull down
Defaults to `100`, the pulled down pixels are equal to the cursor
```

What does it mean to be "equal to the cursor"? Is just number of pixels moved is equal to the cursor position delta?